lib/tasks/data_updates.rake

```diff
@@ -13,4 +13,16 @@ namespace :data_updates do
   task run: :environment do
     DataUpdateWorker.new.perform
   end
+
+  desc "Get data update status"
+  task status: :environment do
+    # rubocop:disable Style/FormatStringToken
+    puts format(" %-9s  %-14s  %-40s", "Status", "ID", "Description")
+    puts "-" * 80
+    DataUpdateScript.find_each do |dus|
```

Comment:
could you silence Rails logging with `ActiveRecord::Base.logger = nil` ?

Otherwise we get:

```
 Status     ID              Description
--------------------------------------------------------------------------------
  DataUpdateScript Load (3.7ms)  SELECT "data_update_scripts".* FROM "data_update_scripts" ORDER BY "data_update_scripts"."id" ASC LIMIT $1  [["LIMIT", 1000]]
  ↳ lib/tasks/data_updates.rake:22:in `block (2 levels) in <main>'
```

in the output

Reply:
Ha, why did I not get this locally? 🤔

Reply:
🤷

Reply:
Anyway, I disabled the logger now as requested.